client/pluginManger.go

```diff
@@ -99,8 +107,12 @@ func (p *PluginManager) GetAvailablePlugins() (pluginList *AvailablePluginList,
 }
 
 // GetPlugins get installed plugins
-func (p *PluginManager) GetPlugins() (pluginList *InstalledPluginList, err error) {
-	err = p.RequestWithData("GET", "/pluginManager/api/json?depth=1", nil, nil, 200, &pluginList)
+func (p *PluginManager) GetPlugins(depth ...int) (pluginList *InstalledPluginList, err error) {
```

Comment:
Why the depth is an array instead int?

Reply:
That's not a reason to use an incorrect parameter.

Reply:
GoLang itself supports variable parameters. Why is it the wrong way?

Reply:
Because you don't use it as an array.

Reply:
If so. Code redundancy is not very good

Reply:
You can give the depth a default value as 1 if it is less than 1. No matter what, you can't use an array which only takes the first element. It does not make sense. And no one can understand why using it like that.